packages/omgx/wallet-frontend/src/components/listFarm/listFarm.js

```diff
@@ -218,7 +218,7 @@ class ListFarm extends React.Component {
 
             <S.GridItemTag item xs={4} md={1.7}>
               {isMobile ? (
-                <Typography variant="overline" sx={{opacity: 0.7}}>APR</Typography>
+                <Typography variant="overline" sx={{opacity: 0.7}}>APR &#x25;</Typography>
```

Comment:
Is there any reason that we have to use hex of % symbol?

Reply:
probably not - please change to %

Reply:
okay